main.py

```diff
@@ -84,6 +86,35 @@ def __init__(self, sqs_apps, elb_apps, scheduled_job_apps):
         self.last_scale_up = {}
         self.last_scale_down = {}
 
+    def load_schedule(self):
+        try:
+            schedule = None
+            with open("schedule.yml") as f:
+                schedule = yaml.load(f)
+        except Exception as e:
+            print(e)
```

Comment:
This will just print to screen? So consider adding logging here or add some guidance in the README on debugging

Reply:
The thinking behind handling the file opening like this is:

1. The `schedule.yml` file will be pushed during the deployment. Since the deployment is happening manually (for now), we should only be deploying from master which is always stable (or at least reviewed).
1. If, despite that, the file cannot be opened for whatever reason (it doesn't exist, it's not a valid yaml, you don't have permission to open it, etc) an exception will be raised. I don't see any benefit in catching specific exceptions, as we cannot do anything about it here, except log the message. However, I agree that we should log it somewhere as this would only appear in the beginning and could be easily lost.